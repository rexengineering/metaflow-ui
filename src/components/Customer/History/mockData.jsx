import { faCommentAltLines } from "@fortawesome/pro-light-svg-icons/faCommentAltLines";
import { faPhoneAlt } from "@fortawesome/pro-light-svg-icons/faPhoneAlt";
import { faCommentAlt } from "@fortawesome/pro-regular-svg-icons/faCommentAlt";

export const CustomerHistory = [
  {
    label: "Call - Connected",
    date: "12/20/19",
    icon: faPhoneAlt,
    color: "#FBAE41",
  },
  {
    label: "Chat - Connected",
    date: "12/20/18",
    icon: faCommentAlt,
    color: "#807DFB",
  },
  {
    label: "Chat - Connected",
    date: "12/20/21",
    icon: faCommentAltLines,
    color: "#FF685F",
  },
];

export const Months = [
  "December 2020",
  "November 2020",
  "October 2020",
  "September 2020",
];
