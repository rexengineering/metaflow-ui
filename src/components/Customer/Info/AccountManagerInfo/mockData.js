export default {
  name: "Duo Maxwell",
  role: "Account Manager",
  phoneNumber: "555-555-5555",
  address: "123 Main Street, Los Angeles CA 90094",
  onTransferCallClick: () => {},
};
