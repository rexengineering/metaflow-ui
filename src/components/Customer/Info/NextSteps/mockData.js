export default ["Schedule an IHC", "Schedule an XXXXX"];
