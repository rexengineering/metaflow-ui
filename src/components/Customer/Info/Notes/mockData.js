export default ["Follow up on home services", "Follow up on sales tips"];
