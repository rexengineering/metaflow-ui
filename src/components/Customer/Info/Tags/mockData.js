export default ["Direct Buyer", "RHL"];
