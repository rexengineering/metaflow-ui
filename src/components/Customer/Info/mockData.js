export const MockedClientInfo = {
  name: "John Doe",
  email: "jdoe@email.com",
  phone: "3105555555",
  mobile: "3105555555",
  type: "Two",
};

export const MockedUserTypes = [
  {
    value: "One",
    label: "One",
  },
  {
    value: "Two",
    label: "Two",
  },
  {
    value: "Three",
    label: "Three",
  },
];
