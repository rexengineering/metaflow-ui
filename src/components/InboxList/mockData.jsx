import { faCommentAltLines } from "@fortawesome/pro-light-svg-icons/faCommentAltLines";
import { faPhoneAlt } from "@fortawesome/pro-light-svg-icons/faPhoneAlt";
import { faCommentAlt } from "@fortawesome/pro-regular-svg-icons/faCommentAlt";

const data = [
  {
    icon: faCommentAlt,
    color: "#FF685F",
    hasShadow: true,
    time: "01:26",
    cellphoneNumber: "(310) - 555 - 5855",
  },
  {
    icon: faCommentAltLines,
    color: "#89C5CC",
    hasShadow: true,
    time: "13:55",
    cellphoneNumber: "(310) - 555 - 5523",
  },
  {
    icon: faPhoneAlt,
    color: "#BDBDBD",
    hasShadow: true,
    time: "14:02",
    cellphoneNumber: "(310) - 555 - 5555",
  },
];

export default data;
