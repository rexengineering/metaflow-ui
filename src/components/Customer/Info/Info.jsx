import {
  Button,
  makeStyles,
  MenuItem,
  Select,
  TextField,
} from "@material-ui/core";
import React from "react";
import {
  string as yupString,
  number as yupNumber,
  object as yupObject,
} from "yup";
import { useFormik } from "formik";
import PropTypes from "prop-types";
import AccountManagerInfo from "./AccountManagerInfo";
import NextSteps from "./NextSteps";
import Notes from "./Notes";
import Tags from "./Tags";
import Address from "./Address";
import {
  customerInfoShape,
  managerInfoShape,
  nextStepsShape,
  notesShape,
  userTypesShape,
  tagsShape,
} from "../../../utils/shapes";

const useStyles = makeStyles((theme) => ({
  container: {
    display: "flex",
    flexDirection: "column",
  },
  margin: {
    margin: theme.spacing(1.5, 0),
  },
}));

const infoValidationSchema = yupObject().shape({
  name: yupString().required(),
  email: yupString().required(),
  phone: yupNumber().required(),
  mobile: yupNumber().required(),
  type: yupString().required(),
});

function Info({
  customerInfo,
  managerInfo,
  nextSteps,
  notes,
  tags,
  userTypes,
  onSubmit,
}) {
  const { name, email, phone, mobile, type } = customerInfo;
  const classes = useStyles();
  const formik = useFormik({
    initialValues: {
      name,
      email,
      phone,
      mobile,
      type,
    },
    onSubmit,
    validationSchema: infoValidationSchema,
  });
  return (
    <section className={classes.container}>
      <form onSubmit={formik.handleSubmit} className={classes.margin}>
        <TextField
          fullWidth
          label="Name"
          className={classes.margin}
          name="name"
          value={formik.values.name}
          error={Boolean(formik.touched.name && formik.errors.name)}
          helperText={formik.touched.name && formik.errors.name}
          onChange={formik.handleChange}
        />
        <TextField
          fullWidth
          label="Email"
          className={classes.margin}
          name="email"
          value={formik.values.email}
          error={Boolean(formik.touched.email && formik.errors.email)}
          helperText={formik.touched.email && formik.errors.email}
          onChange={formik.handleChange}
        />
        <TextField
          fullWidth
          label="Phone"
          className={classes.margin}
          name="phone"
          value={formik.values.phone}
          error={Boolean(formik.touched.phone && formik.errors.phone)}
          helperText={formik.touched.phone && formik.errors.phone}
          onChange={formik.handleChange}
        />
        <TextField
          fullWidth
          label="Mobile"
          className={classes.margin}
          name="mobile"
          value={formik.values.mobile}
          error={Boolean(formik.touched.mobile && formik.errors.mobile)}
          helperText={formik.touched.mobile && formik.errors.mobile}
          onChange={formik.handleChange}
        />
        <Select
          fullWidth
          label="Type"
          variant="filled"
          className={classes.margin}
          name="type"
          value={formik.values.type}
          error={Boolean(formik.touched.type && formik.errors.type)}
          onChange={formik.handleChange}
        >
          {Array.isArray(userTypes) &&
            userTypes.map(({ value, label }) => (
              <MenuItem key={value} value={value}>
                {label}
              </MenuItem>
            ))}
        </Select>
        <Button type="submit" name="submit">
          Save
        </Button>
      </form>
      <Tags className={classes.margin} tagsItems={tags} />
      <Notes className={classes.margin} notes={notes} />
      <NextSteps
        onNextStepsSelected={() => {}}
        onAddNextStepsButtonClicked={() => {}}
        className={classes.margin}
        nextSteps={nextSteps}
      />
      <AccountManagerInfo
        name={managerInfo.name}
        phoneNumber={managerInfo.phoneNumber}
        roleType={managerInfo.role}
        onTransferCallClick={managerInfo.onTransferCallClick}
        className={classes.margin}
      />
      <Address className={classes.margin} address={managerInfo.address} />
    </section>
  );
}

Info.propTypes = {
  customerInfo: customerInfoShape.isRequired,
  managerInfo: managerInfoShape.isRequired,
  nextSteps: nextStepsShape.isRequired,
  notes: notesShape.isRequired,
  userTypes: userTypesShape.isRequired,
  tags: tagsShape.isRequired,
  onSubmit: PropTypes.func.isRequired,
};

export default Info;
