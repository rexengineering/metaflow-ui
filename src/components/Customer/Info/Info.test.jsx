import { fireEvent, render, waitFor } from "@testing-library/react";
import React from "react";
import { MockedClientInfo, MockedUserTypes } from "./mockData";
import MockedManagerInfo from "./AccountManagerInfo/mockData";
import MockedNextStepsInfo from "./NextSteps/mockData";
import MockedNotesInfo from "./Notes/mockData";
import MockedTagsInfo from "./Tags/mockData";
import Info from "./Info";

describe("<Info />", () => {
  const renderInfoComponent = (onInfoSubmit = () => {}) => {
    return (
      <Info
        customerInfo={MockedClientInfo}
        managerInfo={MockedManagerInfo}
        nextSteps={MockedNextStepsInfo}
        notes={MockedNotesInfo}
        tags={MockedTagsInfo}
        onSubmit={onInfoSubmit}
        userTypes={MockedUserTypes}
      />
    );
  };

  it("should render", () => {
    const { asFragment } = render(renderInfoComponent());
    const initialRender = asFragment();

    expect(initialRender).not.toBeNull();
  });

  it("should match snapshot", () => {
    const { asFragment } = render(renderInfoComponent());
    const initialRender = asFragment();

    expect(initialRender).toMatchSnapshot();
  });

  it("should handle data change for form inputs properly", async () => {
    const { container } = render(renderInfoComponent());
    const nameInput = container.querySelector("[name='name']");
    const emailInput = container.querySelector("[name='email']");
    const phoneInput = container.querySelector("[name='phone']");
    const mobileInput = container.querySelector("[name='mobile']");

    await waitFor(() => {
      fireEvent.change(nameInput, { target: { value: MockedClientInfo.name } });
      fireEvent.change(emailInput, {
        target: { value: MockedClientInfo.email },
      });
      fireEvent.change(phoneInput, {
        target: { value: MockedClientInfo.phone },
      });
      fireEvent.change(mobileInput, {
        target: { value: MockedClientInfo.mobile },
      });
    });

    expect(nameInput.value).toBe(MockedClientInfo.name);
    expect(emailInput.value).toBe(MockedClientInfo.email);
    expect(phoneInput.value).toBe(MockedClientInfo.phone);
    expect(mobileInput.value).toBe(MockedClientInfo.mobile);
  });
});
