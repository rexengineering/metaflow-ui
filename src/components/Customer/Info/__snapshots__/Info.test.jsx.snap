// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Info /> should match snapshot 1`] = `
<DocumentFragment>
  <section
    class="makeStyles-container-18"
  >
    <form
      class="makeStyles-margin-19"
    >
      <div
        class="MuiFormControl-root MuiTextField-root makeStyles-margin-19 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
          data-shrink="true"
        >
          Name
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            name="name"
            type="text"
            value="John Doe"
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root makeStyles-margin-19 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
          data-shrink="true"
        >
          Email
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            name="email"
            type="text"
            value="jdoe@email.com"
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root makeStyles-margin-19 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
          data-shrink="true"
        >
          Phone
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            name="phone"
            type="text"
            value="3105555555"
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root makeStyles-margin-19 MuiFormControl-fullWidth"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
          data-shrink="true"
        >
          Mobile
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-fullWidth MuiInput-fullWidth MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            name="mobile"
            type="text"
            value="3105555555"
          />
        </div>
      </div>
      <div
        class="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline makeStyles-margin-19 MuiInputBase-fullWidth"
      >
        <div
          aria-haspopup="listbox"
          aria-labelledby="mui-component-select-type"
          class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-filled MuiInputBase-input MuiFilledInput-input"
          id="mui-component-select-type"
          role="button"
          tabindex="0"
        >
          Two
        </div>
        <input
          aria-hidden="true"
          class="MuiSelect-nativeInput"
          name="type"
          tabindex="-1"
          value="Two"
        />
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconFilled"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-text"
        name="submit"
        tabindex="0"
        type="submit"
      >
        <span
          class="MuiButton-label"
        >
          Save
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </form>
    <section
      class="makeStyles-margin-19"
    >
      <div
        class="makeStyles-header-20"
      >
        <h6
          class="MuiTypography-root makeStyles-label-21 MuiTypography-h6"
        >
          Tags
        </h6>
        <button
          class="MuiButtonBase-root MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-plus-circle fa-w-16 "
              data-icon="plus-circle"
              data-prefix="fal"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M384 250v12c0 6.6-5.4 12-12 12h-98v98c0 6.6-5.4 12-12 12h-12c-6.6 0-12-5.4-12-12v-98h-98c-6.6 0-12-5.4-12-12v-12c0-6.6 5.4-12 12-12h98v-98c0-6.6 5.4-12 12-12h12c6.6 0 12 5.4 12 12v98h98c6.6 0 12 5.4 12 12zm120 6c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zm-32 0c0-119.9-97.3-216-216-216-119.9 0-216 97.3-216 216 0 119.9 97.3 216 216 216 119.9 0 216-97.3 216-216z"
                fill="currentColor"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div>
        <div
          class="makeStyles-container-22"
        >
          <p
            class="MuiTypography-root makeStyles-label-23 MuiTypography-body1"
          >
            Direct Buyer
          </p>
          <div
            class="makeStyles-actionElement-24"
          />
        </div>
        <div
          class="makeStyles-container-22"
        >
          <p
            class="MuiTypography-root makeStyles-label-23 MuiTypography-body1"
          >
            RHL
          </p>
          <div
            class="makeStyles-actionElement-24"
          />
        </div>
      </div>
    </section>
    <section
      class="makeStyles-margin-19"
    >
      <div
        class="makeStyles-header-20"
      >
        <h6
          class="MuiTypography-root makeStyles-label-21 MuiTypography-h6"
        >
          Notes
        </h6>
        <button
          class="MuiButtonBase-root MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-plus-circle fa-w-16 "
              data-icon="plus-circle"
              data-prefix="fal"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M384 250v12c0 6.6-5.4 12-12 12h-98v98c0 6.6-5.4 12-12 12h-12c-6.6 0-12-5.4-12-12v-98h-98c-6.6 0-12-5.4-12-12v-12c0-6.6 5.4-12 12-12h98v-98c0-6.6 5.4-12 12-12h12c6.6 0 12 5.4 12 12v98h98c6.6 0 12 5.4 12 12zm120 6c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zm-32 0c0-119.9-97.3-216-216-216-119.9 0-216 97.3-216 216 0 119.9 97.3 216 216 216 119.9 0 216-97.3 216-216z"
                fill="currentColor"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div>
        <div
          class="makeStyles-container-22"
        >
          <p
            class="MuiTypography-root makeStyles-label-23 MuiTypography-body1"
          >
            Follow up on home services
          </p>
          <div
            class="makeStyles-actionElement-24"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root makeStyles-iconButtonSize-25"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-trash-alt fa-w-14 "
                  data-icon="trash-alt"
                  data-prefix="fal"
                  focusable="false"
                  role="img"
                  viewBox="0 0 448 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M296 432h16a8 8 0 0 0 8-8V152a8 8 0 0 0-8-8h-16a8 8 0 0 0-8 8v272a8 8 0 0 0 8 8zm-160 0h16a8 8 0 0 0 8-8V152a8 8 0 0 0-8-8h-16a8 8 0 0 0-8 8v272a8 8 0 0 0 8 8zM440 64H336l-33.6-44.8A48 48 0 0 0 264 0h-80a48 48 0 0 0-38.4 19.2L112 64H8a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h24v368a48 48 0 0 0 48 48h288a48 48 0 0 0 48-48V96h24a8 8 0 0 0 8-8V72a8 8 0 0 0-8-8zM171.2 38.4A16.1 16.1 0 0 1 184 32h80a16.1 16.1 0 0 1 12.8 6.4L296 64H152zM384 464a16 16 0 0 1-16 16H80a16 16 0 0 1-16-16V96h320zm-168-32h16a8 8 0 0 0 8-8V152a8 8 0 0 0-8-8h-16a8 8 0 0 0-8 8v272a8 8 0 0 0 8 8z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="makeStyles-container-22"
        >
          <p
            class="MuiTypography-root makeStyles-label-23 MuiTypography-body1"
          >
            Follow up on sales tips
          </p>
          <div
            class="makeStyles-actionElement-24"
          >
            <button
              class="MuiButtonBase-root MuiIconButton-root makeStyles-iconButtonSize-25"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-trash-alt fa-w-14 "
                  data-icon="trash-alt"
                  data-prefix="fal"
                  focusable="false"
                  role="img"
                  viewBox="0 0 448 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M296 432h16a8 8 0 0 0 8-8V152a8 8 0 0 0-8-8h-16a8 8 0 0 0-8 8v272a8 8 0 0 0 8 8zm-160 0h16a8 8 0 0 0 8-8V152a8 8 0 0 0-8-8h-16a8 8 0 0 0-8 8v272a8 8 0 0 0 8 8zM440 64H336l-33.6-44.8A48 48 0 0 0 264 0h-80a48 48 0 0 0-38.4 19.2L112 64H8a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h24v368a48 48 0 0 0 48 48h288a48 48 0 0 0 48-48V96h24a8 8 0 0 0 8-8V72a8 8 0 0 0-8-8zM171.2 38.4A16.1 16.1 0 0 1 184 32h80a16.1 16.1 0 0 1 12.8 6.4L296 64H152zM384 464a16 16 0 0 1-16 16H80a16 16 0 0 1-16-16V96h320zm-168-32h16a8 8 0 0 0 8-8V152a8 8 0 0 0-8-8h-16a8 8 0 0 0-8 8v272a8 8 0 0 0 8 8z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </section>
    <section
      class="makeStyles-margin-19"
    >
      <div
        class="makeStyles-header-20"
      >
        <h6
          class="MuiTypography-root makeStyles-label-21 MuiTypography-h6"
        >
          Next Steps
        </h6>
        <button
          class="MuiButtonBase-root MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-plus-circle fa-w-16 "
              data-icon="plus-circle"
              data-prefix="fal"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M384 250v12c0 6.6-5.4 12-12 12h-98v98c0 6.6-5.4 12-12 12h-12c-6.6 0-12-5.4-12-12v-98h-98c-6.6 0-12-5.4-12-12v-12c0-6.6 5.4-12 12-12h98v-98c0-6.6 5.4-12 12-12h12c6.6 0 12 5.4 12 12v98h98c6.6 0 12 5.4 12 12zm120 6c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zm-32 0c0-119.9-97.3-216-216-216-119.9 0-216 97.3-216 216 0 119.9 97.3 216 216 216 119.9 0 216-97.3 216-216z"
                fill="currentColor"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
      <div>
        <div
          class="makeStyles-container-22"
        >
          <p
            class="MuiTypography-root makeStyles-label-23 MuiTypography-body1"
          >
            Schedule an IHC
          </p>
          <div
            class="makeStyles-actionElement-24"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-26 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-29"
                  data-indeterminate="false"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
          </div>
        </div>
        <div
          class="makeStyles-container-22"
        >
          <p
            class="MuiTypography-root makeStyles-label-23 MuiTypography-body1"
          >
            Schedule an XXXXX
          </p>
          <div
            class="makeStyles-actionElement-24"
          >
            <span
              aria-disabled="false"
              class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-26 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
            >
              <span
                class="MuiIconButton-label"
              >
                <input
                  class="PrivateSwitchBase-input-29"
                  data-indeterminate="false"
                  type="checkbox"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </span>
          </div>
        </div>
      </div>
    </section>
    <section
      class="makeStyles-fontColor-32 makeStyles-margin-19"
    >
      <div
        class="makeStyles-header-20"
      >
        <h6
          class="MuiTypography-root makeStyles-label-21 MuiTypography-h6"
        >
          Team
        </h6>
      </div>
      <div>
        <div
          class="makeStyles-managerInfoContainer-30"
        >
          <div
            class="MuiAvatar-root MuiAvatar-circle makeStyles-avatar-31 MuiAvatar-colorDefault"
          >
            D
          </div>
          <div>
            <p
              class="MuiTypography-root makeStyles-accountManagerName-33 MuiTypography-body1"
            >
              Duo Maxwell
            </p>
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              Account Manager
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              555-555-5555
            </h6>
          </div>
        </div>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-transferCallButton-34 MuiButton-fullWidth"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Transfer Call
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </section>
    <section
      class="makeStyles-margin-19"
    >
      <div
        class="makeStyles-header-20"
      >
        <h6
          class="MuiTypography-root makeStyles-label-21 MuiTypography-h6"
        >
          Address
        </h6>
      </div>
      <div>
        <h6
          class="MuiTypography-root MuiTypography-h6"
        >
          123 Main Street, Los Angeles CA 90094
        </h6>
      </div>
    </section>
  </section>
</DocumentFragment>
`;
